### PR TITLE
Feature/3.2/event service testability

### PR DESCRIPTION
**Pull request by cklellie**
Changes for testability of EventServer.

- Added `withStart` and `withEnd` to generate fake events.
- Moved time caluations into existing InstantUtil.
- Added configuration for EventServer shutdown, so default of 2 second quietPeriod is not longer mandatory when cleaning up during unit testing.

### CODE REVIEW

File: casual/casual-api/src/main/java/se/laz/casual/config/Configuration.java
Comment (aliothe) [inline casual/casual-api/src/main/java/se/laz/casual/config/Configuration.java:1]: Why this change?

File: casual/casual-api/src/main/java/se/laz/casual/config/Shutdown.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021, The casual project. All rights reserved.
```
Comment (aliothe) [inline casual/casual-api/src/main/java/se/laz/casual/config/Shutdown.java]: 2024 no? :)

**Review by aliothe — APPROVED** [review]
Looks good besides 2021 instead of 2024 and I am curious as to why the toString method in Configuration changed to using the attributes instead of the methods?

File: casual/casual-api/src/main/java/se/laz/casual/config/Configuration.java
Comment (cklellie) [reply casual/casual-api/src/main/java/se/laz/casual/config/Configuration.java:1]: I regenerated it to include event server, i missed it used methods not attributes. I'll change back to methods again.

File: casual/casual-event-server/src/test/groovy/se/laz/casual/event/server/EventServerConnectionInformationTest.groovy
Comment (TobiasLeo) [inline casual/casual-event-server/src/test/groovy/se/laz/casual/event/server/EventServerConnectionInformationTest.groovy:1]: License header?
